app/controllers/reactions_controller.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class ReactionsController < ApplicationController
+  skip_before_action :verify_authenticity_token
```

Comment:
Not sure if I should leave this in or remove it. I added it because I thought it would make things easier?

Reply:
I'm not sure what `verify_authenticity_token` checks here - does it include the CSRF token? If so I'd be in favour of skipping it for interview purposes. I imagine we're looking to submit via JS (i.e. without assistance from rails form helpers) so might require extra fiddling to get it working.

Reply:
Yup it is the CSRF token! I added the skip because it was a pain when I was trying to hack up a front end integration 😬